src/base/Group.h

```diff
@@ -31,6 +71,17 @@ class Group : private std::set<T*>
         return parent::end();
     }
 
+    using parent::size;
+    using parent::insert;
```

Comment:
Is `insert` still necessary? It would mess up all the sorting if someone were to use `insert` instead of `Add`, right?